.pylintrc

```diff
@@ -64,7 +64,8 @@ disable=missing-docstring,
       wrong-import-order, # Leave this up to isort
       bad-continuation, # Leave this up to black
       line-too-long, # Leave this up to black
-      exec-used
+      exec-used,
+      duplicate-code
```

Comment:
I tried disabling this check at the file level, but pylint ignored it :( the duplicate code was tests which was duplicated was between separate unit tests